docs/reduction_toolbox.rst

```diff
@@ -99,6 +99,22 @@ Use this technique with `~ccdproc.cosmicray_lacosmic`:
 
     >>> cr_cleaned = ccdproc.cosmicray_lacosmic(gain_corrected, sigclip=5)
 
+.. note::
+
+    A significant error in `~ccdproc.cosmicray_lacosmic` in version 2.1 of
```

Comment:
I think I'd just remove the first line here, but I think the rest gets across the point.